applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_server.c

```diff
@@ -44,9 +46,9 @@ static enum audio_channel channel;
 /* Advertising data for peer connection */
 static uint8_t csip_rsi_adv_data[BT_CSIP_RSI_SIZE];
 
-static uint8_t flags_adv_data[] = {BT_LE_AD_GENERAL | BT_LE_AD_NO_BREDR};
+static uint8_t flags_adv_data[1];
```

Comment:
A tiny bit magic. Can we define or explain "1"?

Reply:
Does not need to be an array.